src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -207,13 +207,21 @@ public async Task GetContentAsync_NullResponse_Throws()
         }
 
         [Fact]
-        [ActiveIssue(28882, TargetFrameworkMonikers.NetFramework)]
         public async Task GetContentAsync_NullResponseContent_ReturnsDefaultValue()
         {
             using (var client = new HttpClient(new CustomResponseHandler((r,c) => Task.FromResult(new HttpResponseMessage() { Content = null }))))
             {
                 Assert.Same(string.Empty, await client.GetStringAsync(CreateFakeUri()));
-                Assert.Same(Array.Empty<byte>(), await client.GetByteArrayAsync(CreateFakeUri()));
+
+                if (PlatformDetection.IsFullFramework)
+                {
+                    Assert.Equal(Array.Empty<byte>(), await client.GetByteArrayAsync(CreateFakeUri()));
+                }
+                else
+                {
+                    Assert.Same(Array.Empty<byte>(), await client.GetByteArrayAsync(CreateFakeUri()));
+                }
```

Comment:
Nit: just to help highlight what's the same and what's different, might have been helpful if the common code were factored out, e.g.
```C#
byte[] actual = await client.GetByteArrayAsync(CreateFakeUri());
if (PlatformDetection.IsFullFramework)
{
    Assert.Equal(Array.Empty<byte>(), actual);
}
else
{
    Assert.Same(Array.Empty<byte>(), actual);
}
```